packages/forma-36-react-components/src/components/Flex/Flex.tsx

```diff
@@ -0,0 +1,221 @@
+import React from 'react';
+import * as CSS from 'csstype';
+import tokens from '@contentful/forma-36-tokens';
+import cn from 'classnames';
+
+import styles from './Flex.css';
+
+export type SpacingTypes = 'none' | 'spacing2xs' | 'spacingXs' | 'spacingS' | 'spacingM' | 'spacingL' | 'spacingXl' | 'spacing2Xl' | 'spacing3Xl' | 'spacing4Xl';
+
+export type FlexContainerType =
```

Comment:
@mshaaban0 I declared then over here. WDYT?

Reply:
@burakukula yeah I noticed, it doesn't include all properties though, see [full html elements list](https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.htmlelementtagnamemap.html), we will have to include the [SVG elements](https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.svgelementtagnamemap.html) as well. imo, it's a lot, and a string would do.

Reply:
@mshaaban0 aaa, sorry, didn't get that. 🤦‍♀️ sure, I can change it to string. I made a selection of them, that's true. hmm. if we would need all of them, then yeah, maybe it's better to be string. Although with the list there would be some suggestions, which might be useful?

Reply:
@mshaaban0 updated ✅

Reply:
@mshaaban0 I think it has to be `any` in that case. in case of `string` we get "Property 'children' does not exist on type 'IntrinsicAttributes'." . I think this is how we handle this in Tooltip, but maybe there is  better way.. not sure. WDYT?

Reply:
@burakukula `any` also would do, thank you for doing the change, I think we just remove `FlexContainerType` type because it's not used anymore.